app/serializers/active_model/errors_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  class ErrorsSerializer
+    def initialize(errors)
+      @errors = errors
+    end
+
+    def serializable_hash
```

Comment:
Maybe we can try to rename `serializeable_hash` to `as_json`, then controller call would be simple.

```
  def as_json
     { errors: @errors.errors.map { |e| { detail: e.full_message, source: e.attribute } } }
  end
```

```
   render json: ActiveModel:: ErrorsSerializer.new(user.errors)
```

Above change is easy, if you want to go step further (optional), you can even override ActiveModel::Errors class https://apidock.com/rails/ActiveModel/Errors/as_json. Then the controller call will be as simple as:

```
   render json: user.errors
```

Reply:
🔥🤯🔥
My head when I finaly understood that render json:  will call the "as_json" method 🙃

I'll go for the first [simpler] option you mentioned :)

Reply:
Done. Any comment is welcome @ankitkalia1195  :)